lib/js/components/TextGroup/TextGroup.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+	<div
+		class="ds-textGroup"
+		:class="{
+			'-ds-small': size === TEXT_GROUP_SIZES.SMALL,
+
+			'-ds-hovered': state === TEXT_GROUP_STATES.HOVERED,
+			'-ds-loading': isLoading,
+			'-ds-disabled': state === TEXT_GROUP_STATES.DISABLED,
+
+			'-ds-interactive': isInteractive,
+			'-ds-selected': isSelected,
+
+			[loadingSizeClassName]: isLoading,
+
+			[colorClassName]: true,
+		}"
+	>
+		<div
+			v-if="eyebrowText !== null"
+			class="ds-textGroup__eyebrow"
+			:class="{
+				'-ds-ellipsis': eyebrowTextEllipsis,
+				'-ds-uppercase': isEyebrowTextUppercase,
+			}"
+		>
+			<ds-skeleton v-if="isLoading" width="50%" height="100%" />
```

Comment:
What if `eyebrowText` is constant, but we need to wait for `mainText` to be loaded? Should we display a skeleton in such case or eyebrow should be displayed?

Reply:
https://bethinkteam.slack.com/archives/CFBLF5RGA/p1723103554903989